### DISCUSSION TITLE
Hope you don.t mind me asking there boards are available to buy

### DISCUSSION THREAD

**Discussion by mutantxxx** [RonMcKay/capacitive-soil-moisture-sensor#2] [Q&A]
Hope you don.t mind me asking the boards are available to buy I can see that
But that at present they  do not have the ATmega328P soldered on 
would there be a place were I purchase with components or most component's on the board ?, 
Not that I am lazy Eyesite is going a bit  .
Happy to pay

**Comment by RonMcKay** [comment]
Hello,

the smt assembly is still something I need to figure out and is still on my todo list.
I am however not quite sure how e.g. [openhardware.io](openhardware.io) handles only partly assembled boards. As a buyer I would be frustrated if I buy an "assembled" board but get half of it as lose parts. Another problem is the availability of the components I used for the board. Especially the through hole battery contacts I haven't seen yet in the shops of the mayor PCB assembly providers.
I might put together some kind of "20% DIY" kit where only a few components have to be soldered but that takes some time and I plan on looking into that when I have some spare time left.

I hope that answers your question. If you have any suggestions for the assembly feel free to share your ideas.

Cheers,
Ron
